tests/test_warm_reboot.py

```diff
@@ -437,7 +437,8 @@ def test_VlanMgrdWarmRestart(self, dvs, testlog):
         (status, fvs) = tbl.get("Vlan20:11.0.0.11")
         assert status == True
 
-        swss_app_check_RestoreCount_single(state_db, restore_count, "vlanmgrd")
+        swss_check_RestoreCount(dvs, state_db, restore_count)
```

Comment:
@qiluo-msft , could you review this?

Reply:
lgtm

Reply:
there are more dependencies on the Reconciled state in the vs tests, I will check and fix.